Mathlib/Data/Equiv/Basic.lean

```diff
@@ -20,11 +20,11 @@ open Function
 variable {α : Sort u} {β : Sort v} {γ : Sort w}
 
 /-- `α ≃ β` is the type of functions from `α → β` with a two-sided inverse. -/
-structure equiv (α : Sort u) (β : Sort v) :=
-(to_fun    : α → β)
-(inv_fun   : β → α)
-(left_inv  : left_inverse inv_fun to_fun)
-(right_inv : right_inverse inv_fun to_fun)
+structure equiv (α : Sort u) (β : Sort v) where
+  to_fun    : α → β
+  inv_fun   : β → α
+  left_inv  : left_inverse inv_fun to_fun
+  right_inv : right_inverse inv_fun to_fun
```

Comment:
```suggestion
structure Equiv (α : Sort u) (β : Sort v) where
  toFun    : α → β
  invFun   : β → α
  leftInv  : left_inverse invFun toFun
  rightInv : right_inverse invFun toFun
```

We might change it to the correct naming convention while you're already looking at the file.